lib/rss.js

```diff
@@ -0,0 +1,191 @@
+const RSS = require('rss')
+const generateDiff = require('deep-diff').diff
+
+function createLeadersHash(array) {
+  return array.reduce((hash, elem) => {
+    hash[elem.id] = elem
+    return hash
+  }, {})
+}
+
+function createOrgHash(array) {
+  let rank = 0
+  return array.reduce((hash, elem) => {
+    rank++
+    elem.rank = rank
+    elem.leaders = createLeadersHash(elem.leaders)
+    hash[elem.slug] = elem
+    return hash
+  }, {})
+}
+
+module.exports = async ({
+  data: newData,
+  data_old: oldData,
+  feed_items: current,
+  data_updated: dataUpdated,
+}) => {
+  return new Promise(resolve => {
```

Comment:
Is there any reason for using Promise?

It's an async function already, should return Promise all the time.

But really, you don't need to be asynchronous in this case and it shouldn't return Promise ..

Reply:
Yep, you're right. I mistakingly thought we were ending/writing inside a callback.